src/main/java/rest/demo/controller/TaskController.java

```diff
@@ -1,8 +1,9 @@
-package co.uk.marcin.kulik.spring.boot.rest.demo.controller;
+package rest.demo.controller;

```

Comment:
Don't need a controller for this, if not being invoked from outside...

Reply:
What do you mean?